modules/gitops-release-notes-1-11-4.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="release-notes-for-gitops-1-11-4_{context}"]
+= Release Notes for {gitops-title} 1.11.4
+
+{gitops-title} 1.11.4 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-11-4_{context}"]
+== Errata updates
+
+[id="rhsa-XXXX-NNNN-gitops-1-11-4-security-update-advisory_{context}"]
+=== RHSA-XXXX:NNNN - {gitops-title} 1.11.4 security update advisory
+
+Issued: 2024-04-08
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-XXXX:NNNN[RHSA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-operators
+----
+
+[id="fixed-issues-1-11-4_{context}"]
+== Fixed issues
+
+* Before this update, users could not use the `argocd-k8s-auth` binary to add GKE, and EKS clusters as this binary was not available in the GitOps container. This update fixes the issue as the `argocd-k8s-auth` binary has been added in the GitOps container. link:https://issues.redhat.com/browse/GITOPS-4226[GITOPS-4226]
+
+* Before this update, when Argo CD attempted to connect to Azure DevOps, an error was displayed due to the deprecation of the `rsa-ssh` host key algorithm, by the Azure DevOps Repository service. This update fixes the issue as Argo CD provides support for the `rsa-ssh` host key algorithms during the communication process between Argo CD and Azure DevOps Repository service. link:https://issues.redhat.com/browse/GITOPS-4543[GITOPS-4543]
+
+* Before this update, the `ignoreDifferences` sync option in Argo CD did not work for array fields. This update fixes the issue by modifying the merge strategy of the `ignoreDifferences` sync option used in the upstream project to handle array fields. As a result, the sync option now functions correctly by allowing users to ignore specific elements in the array during sync. link:https://issues.redhat.com/browse/GITOPS-2962[GITOPS-2962]
+
+* Before this update, users accessing the ROSA cluster after hibernation were unable to login to the Argo CD Web Console UI due to an error indicating an invalid redirect URI in the Dex configuration. To resolve this issue, users had to restart the {gitops-title} Operator pod along with the Server and Dex pods of Argo CD. With this update, users can now log in to the Argo CD Web Console without facing any errors once the ROSA cluster is operational post-hibernation. link:https://issues.redhat.com/browse/GITOPS-4358[GITOPS-4358]
```

Comment:
```suggestion
* Before this update, users accessing the ROSA cluster after hibernation were unable to log in to the Argo CD web console due to an error indicating an invalid redirect URI in the Dex configuration. With this update, users can now log in to the Argo CD web console without facing any errors when the ROSA cluster is operational post-hibernation. link:https://issues.redhat.com/browse/GITOPS-4358[GITOPS-4358]
```

Reply:
Also, I removed the previous workaround as it is irrelevant now when it is fixed.

Reply:
Fixed.